src/components/OrderSidebar.js

```diff
@@ -78,12 +89,15 @@ const OrderSidebar = () => {
                   secondary={`Artist: ${item.album.artistName}, Price: $${item.album.price}`}
                 />
               </ListItem>
-              <ListItem>
+              <div>
```

Comment:
Any particular issue that led to this element change?

Reply:
No, this is just now, later we will move to the cart page and add style, we will remove this div. Now the button is on the text, without div